apps/builder/app/shared/copy-paste/plugin-markdown.test.ts

```diff
@@ -0,0 +1,128 @@
+import { describe, expect } from "@jest/globals";
+import { Instance, Prop } from "@webstudio-is/project-build";
+import { parse } from "./plugin-markdown";
+
+const parseInstanceData = (data?: {
+  instances: Instance["children"];
+  props: Array<Prop>;
+}) => {
+  if (data === undefined) {
+    return;
+  }
+  const { instances, props } = data;
+  for (const instance of instances) {
+    if ("id" in instance) {
+      Instance.parse(instance);
+    }
+    if ("children" in instance) {
+      parseInstanceData({ instances: instance.children, props });
+    }
+  }
+  for (const prop of props) {
+    Prop.parse(prop);
+  }
+  return { instances, props };
+};
+
+const options = { generateId: () => "123" };
+
+describe("Plugin Markdown", () => {
+  test("paragraph", () => {
+    expect(parseInstanceData(parse("xyz", options))).toMatchSnapshot();
```

Comment:
Uncolocated snapshots not very readable

Reply:
I started with inline snapshots but they are quite big, the entire test becomes to big, was hard to navigate too, inline is better when snapshots are small

Reply:
moved them anyway